core-war/src/main/webapp/attachment/jsp/displayAttachedFiles.jsp

```diff
@@ -87,7 +88,9 @@
   <view:settings var="onlineEditingWithCustomProtocolAlert" settings="org.silverpeas.util.attachment.Attachment" defaultValue="${true}" key="attachment.onlineEditing.customProtocol.alert" />
   <c:set var="webdavEditingEnable" value="${mainSessionController.webDAVEditingEnabled && onlineEditingEnable}" />
```

Comment:
Shouldn't the WebDAV editing be disabled if a huge processing is currently done on attachments?

Reply:
The WebDAV is indeed disabled if a huge processing is currently done on attachments.
This variable is here given to the foolowing tag:
```
        <menu:simpleDocument attachment="${currentAttachment}"
                             showMenuNotif="${showMenuNotif}"
                             useWebDAV="${webdavEditingEnable}"
                             useXMLForm="${useXMLForm}"
                             fromAlias="${fromAlias}"
                             userRole="${highestUserRole}"/>
 ```
The webdav functionnalities are disabled from this tag.

Reply:
ok